services/blogs/src/index.ts

```diff
@@ -0,0 +1,74 @@
+import express from 'express'
+import cors from 'cors'
+
+const app = express()
+const PORT = 3003
+app.use(cors())
+app.use(express.json())
+
+app.get('/api/hackmd', async (req, res) => {
+  const token = '4DXXJDRZGGRFTYTV1LH633U66W32PGKTKF6NPEQKPGNEO5TJXJ'
```

Comment:
If this is a private token, we'll need to create a new HackMD account and throw this in an environment variable.

Reply:
need `.env` variable for this?